connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -1117,41 +1063,27 @@ async function findResourcesNotSeenInGarbageCollectionRun(
     return { pageIdsSeenInRun, databaseIdsSeenInRun };
   })();
 
-  // Temp: debug logging for a customer issue
-  const duplicatePageId = "167582ad-8d56-4f2d-a90c-bdb1289e55b6";
-  if (pageIdsSeenInRun.has(duplicatePageId)) {
-    logger.info(
-      {
-        pageIdsSeenInRun: pageIdsSeenInRun.entries(),
-      },
-      `${duplicatePageId} has been seen in this run`
-    );
-  }
-
-  const pageSize = 500;
-
   const pagesNotSeenInGarbageCollectionRun: Array<{
     lastSeenTs: Date;
     resourceType: "page";
     resourceId: string;
-    skipReason: string | null;
   }> = [];
   const pages = (
     await NotionPage.findAll({
       where: {
         connectorId,
+        skipReason: {
```

Comment:
Just making sure: we have an index on this right ?

Reply:
Nope, only connectorId, but it seems to be very fast, even on our largest customer.